ui/js/directives/treeherder/clonejobs.js

```diff
@@ -127,7 +127,6 @@ treeherder.directive('thCloneJobs', [
         var selectJob = function(job, job_selection_type) {
             var jobKey = getJobMapKey(job);
             var jobEl = $('.' + jobKey);
-
             clickJobCb({}, jobEl, job, job_selection_type);
             scrollToElement(jobEl);
```

Comment:
I don't think we want to emit a signal that the selected job has changed if we're just re-adding the selecting job styles after an update. Likewise I don't think we want to re-scroll to the selected job.